src/emqx_sys.erl

```diff
@@ -87,6 +88,10 @@ stop() ->
 -spec(version() -> string()).
 version() -> emqx_app:get_release().
 
+%% @doc Get cluster name
```

Comment:
nit: remove unnecessary comments

Reply:
For same style as other apis.
like `version/0` `uptime/0` `sysdescr/0`